docs/userguide/campaigns.md

```diff
@@ -1,25 +1,140 @@
\ No newline at end of file
+campaign = Campaign(
+    searchspace=my_searchspace,  # Required
+    objective=my_objective,  # Required
+    strategy=my_strategy,  # Optional
+    numerical_measurements_must_be_within_tolerance=my_boolean,  # Optional
+)
+~~~
+
+```{attention}
+Note that we currently also expose other fields via the constructor. This is
+only temporary, and the corresponding fields should be ignored.
+```
+
+### Creating a campaign via a JSON config
+
+It is also possible to specify a `Campaign` via a configuration string and using the
+function [`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+The specification of config files as well as what they contain are automatically
+derived from the class structure.
+Furthermore, a config file can be validated using 
+[Campaign.validate_config](baybe.campaign.Campaign.validate_config).
+For more details and a full exemplary config, we refer to the corresponding
+[example](./../../examples/Serialization/create_from_config).
+
+## Getting recommendations
+
+### Basics
+
+```{attention}
+Obtaining recommendations resp. adding measurements using the `recommend` resp.
+`add_measurements` functions is the only safe way to inform a `Campaign` object about
+new measurements. These functions update the necessary metadata that is crucial for the
+proper execution of a campaign. It is important to rely on these functions to maintain
+the integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+the `batch_quantity` keyword, specifying the desired size of
+the batch of experiments to be conducted.
+
+~~~python
+rec = campaign.recommend(batch_quantity=3)
+~~~
+
+The `recommend` function returns a `DataFrame` with `batch_quantity` many rows, each 
+representing a set of parameters from the search space.
+
+```{important}
+There is a difference between performing multiple recommendations
+with batch size of 1 and a single recommendation with a larger batch size.
+* **Batch size larger than 1**: The recommended experiments
+are chosen to *jointly* optimized the acquisition function.
+This means that the recommendations are made considering the interaction of multiple
+experiments together.
+* **Batch size of 1**: When making several smaller recommendations instead, each
+*individual* recommendation optimizes the acquisition function at the specific point in 
+time when it is requested. In this case, the recommendations are made independently of
+each other without considering the joint optimization.
+
+Note that this distinction might not be relevant or applicable for all possible
+situation, for example when using recommenders that do not perform joint optimization.
```

Comment:
What was the solution that we agreed upon here? Should we list everything explicitly? Not that critical here imo since users still *can* use arbitrary recommenders here.
Which recommenders that we have actually perform joint optimization?

Reply:
Added a corresponding comment, please resolve if happy.